packages/css/src/components/alert/alert.scss

```
@@ -13,6 +13,9 @@
 
 .ams-alert__severity-indicator {
   background-color: var(--ams-alert-severity-indicator-background-color);
+  border-color: var(--ams-alert-border-color);
```

I think you can inherit the border color.
```suggestion
  border-color: inherit;
```

---

Done.